### PR TITLE
update dependent helm chart matrix

### PR DESCRIPTION
**Pull request by vsinghal13**
###### Description

image

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Review by pmalek-sumo — APPROVED** [review]

**Comment by perk-sumo** [comment]
There is a [PR #958 for versions bump](https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/958), let's wait until it's merged/closed.

**Pull request merged**
